msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/stress/TestCaseStress.java

```diff
@@ -0,0 +1,136 @@
+
+    private String accessToken = null;
+    private Date expiresAt = null;
+
+
+    @Test
+    public void test_acquireTokenSilentlyWithCachedTokens() throws Exception {
+        run();
+    }
+
+    @Override
+    public boolean isTestPassed(IAuthenticationResult result) {
+        if (result == null) {
+            return false;
+        }
+
+        if (accessToken == null || expiresAt.before(new Date())) {
+            accessToken = result.getAccessToken();
+            expiresAt = result.getExpiresOn();
+        }
+
+        return accessToken.equals(result.getAccessToken());
+    }
+
+    @Override
+    public IAccount prepare() {
+        final TokenRequestLatch tokenRequestLatch = new TokenRequestLatch(1);
+
+        final AcquireTokenParameters parameters = new AcquireTokenParameters.Builder()
+                .startAuthorizationFromActivity(mActivity)
+                .withLoginHint(mLoginHint)
+                .withScopes(Arrays.asList(mScopes))
+                .withCallback(successfulInteractiveCallback(tokenRequestLatch))
+                .withPrompt(Prompt.SELECT_ACCOUNT)
+                .build();
+
+        final InteractiveRequest interactiveRequest = new InteractiveRequest(
+                mApplication,
+                parameters,
+                new OnInteractionRequired() {
+                    @Override
+                    public void handleUserInteraction() {
+                        final String username = mLoginHint;
+                        final String password = LabConfig.getCurrentLabConfig().getLabUserPassword();
+
+                        final PromptHandlerParameters promptHandlerParameters = PromptHandlerParameters.builder()
+                                .prompt(PromptParameter.SELECT_ACCOUNT)
+                                .loginHint(mLoginHint)
+                                .sessionExpected(false)
+                                .consentPageExpected(false)
+                                .speedBumpExpected(false)
+                                .build();
+
+                        new AadPromptHandler(promptHandlerParameters).handlePrompt(username, password);
+                    }
+                }
+        );
+
+        interactiveRequest.execute();
+        tokenRequestLatch.await(TokenRequestTimeout.SHORT);
+
+        return getAccount();
+    }
+
+    @Override
+    public IAuthenticationResult execute(final IAccount account) throws Exception {
+        final AcquireTokenSilentParameters acquireTokenParameters = new AcquireTokenSilentParameters.Builder()
+                .forAccount(account)
+                .fromAuthority(account.getAuthority())
+                .forceRefresh(false)
+                .withScopes(Arrays.asList(mScopes))
+                .build();
+
+        return mApplication.acquireTokenSilent(acquireTokenParameters);
+    }
+
+    @Override
+    public int getNumberOfThreads() {
+        return 10;
```

Comment:
I mean, any more than the min of the number of CPUs in the device and the number of threads in the service is going to be kind of useless, and even that isn't going to tell as much as it could, given that your threads are also running.